test/e2e/ingress.go

```diff
@@ -21,6 +21,10 @@ import (
 	"path/filepath"
 	"time"
 
+	legacyv1 "k8s.io/kubernetes/pkg/api/v1"
```

Comment:
Am not against legacyv1 or apiv1. Just a question of consistency throughout the codebase. Feel free to rename it everywhere.